### ISSUE TITLE
Make a list of questions we want answered in the readme of the ios-assignment

### ISSUE THREAD

**Issue by marijnschilling** [bakkenbaeck/ios-workshop#16]
It will be good to have some guidelines about what we would like answered in the readme.

**Comment by roberthein** [comment]
- Can you name the part(s) in your code or design you are more most proud of?
- Can you name the part(s) in your code or design you are more least proud of?
- What would you add or impove if you had more time?

**Comment by elland** [comment]
1. What did you think?  
We’re always looking to improve our process, was there anything you didn’t like or that wasn’t clear? Anything you enjoyed in particular?
2. Why did you choose this particular architecture? Easier to use? Faster to prototype for a one-day project? Wanted to try something new?
3. What design/architectural choices that you made during the assignment were challenging for you? In a good way? In a bad way?
4. If this were a real project, what would you have liked to do next?
5. Dependencies: if they used/didn't use them and why.

**Comment by elland** [comment]
From mine I think we can leave 1. ,2. and 3. for the post assignment talk. Mine 4. and @roberthein's last are more or less the same, we can make that one question. So I vote for @roberthein's questions + my last question.

**Comment by marijnschilling** [comment]
Yeah I agree! Good questions to ask them what they are proud of, very open :) 

so then we have:
1. Can you name the part(s) in your code or design you are more most proud of?
2. Can you name the part(s) in your code or design you are least proud of?
3. What would you add or improve if you had more time?
4. Explain why you chose to use the dependencies you used, or why you chose not to use any.
